<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Receive Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseFileButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse a File</string>
   </property>
  </widget>
  <widget class="QLabel" name="decryptionLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Decryption status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="verificationLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Verification status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="decryptionEmptyLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>81</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="verificationEmptyLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Author:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>361</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveFileButton">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>302</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save a File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
